src/Components/App/App.js

```diff
@@ -0,0 +1,107 @@
+import { Component } from 'react';
+import { getRandomFiveLetterWord, findWordInAPIDatabase } from '../../apiCalls';
+import Header from '../Header/Header';
+import GameBoardContainer from '../GameBoardContainer/GameBoardContainer';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      currentWordInPlay: null,
+      typedLetters: [],
+      submittedWords: [],
+      gameOver: false,
+      playerStats: [],
+      error: null
+    }
+  }
+
+  componentDidMount = () => {
+    return getRandomFiveLetterWord()
+      .then(data => this.setState({ currentWordInPlay: data.word }))
+      .catch(error => {
+        this.setState({ error: error.message })});
+  }
+
+  typeLetter = (letter) => {
+    if (this.state.typedLetters.length < 5 && !this.state.gameOver) {
+      const updatedTypedLetters = [...this.state.typedLetters, letter];
+      this.setState({ typedLetters: updatedTypedLetters });
+    }
+  }
```

Comment:
I know *of* Wordle, but I don't know how to play.  Is it always a 5 letter word?

Reply:
Yes, the way that I've set it up you'll always get a 5 letter word from the API.